frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/__tests__/PipelineRunCount.spec.tsx

```diff
@@ -58,6 +58,8 @@ describe('Pipeline Run Count Graph', () => {
   it('Should render an TimeSeriesChart if data is available', () => {
     pipelineRunSpy.mockReturnValue([{ data: { result: [{ x: 'x' }] } }, false, false]);
     const PipelineRunCountWrapper = shallow(<PipelineRunCount {...PipelineRunCountProps} />);
-    expect(PipelineRunCountWrapper.find(TimeSeriesChart).exists()).toBe(true);
+    expect(PipelineRunCountWrapper.find(LoadingInline).exists()).toBe(false);
+    expect(PipelineRunCountWrapper.find(GraphEmpty).exists()).toBe(false);
+    expect(PipelineRunCountWrapper.find(Measure).exists()).toBe(true);
```

Comment:
@karthikjeeyar Is this change really necessary? The change of this tests "... if data is available" doesn't check that there is a TimeSeriesChart. I would like to see that this test case still tests for the TimeSeriesChart, or? Wdyt?

Reply:
@karthikjeeyar looks like you need to go two levels deeper with the shallow-renderer to find your TimeSeriesChart again. One for the `<Measure />` and one for the children-render-function:

```
    expect(
      PipelineRunCountWrapper.dive()
        .dive()
        .find(TimeSeriesChart)
        .exists(),
    ).toBe(true);
```

Want add this test again? Up to you if you want also keep the other checks. With the Measure check you test some internal which are not required in my opionion.

Reply:
Agreed @jerolimov. I will update the tests and push again. Thanks for pointing out.

Reply:
Updated.